composer.json

```diff
@@ -47,7 +47,12 @@
     },
     "config": {
         "optimize-autoloader": true,
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "composer/package-versions-deprecated": true,
```

Comment:
Given that we are on 8.0+, this can be dropped, as we're supposed to use the new composer-plugin-api anyway